### ISSUE TITLE
Peerage of Science

### ISSUE THREAD

**Issue by nickschurch** [sciencefair-land/datasource-requests#7]
Peer review prior to publication site - it'd be cood to get as many peer reviews included alongside articles as possible I reckon.

**Comment by blahah** [comment]
Yes - in general we will distinguish between **article** datasources and **enrichment** datasources - the latter will include peer review, metrics etc. Article datasources will take priority for now as that is the focus for v1.1 (see [roadmap](https://github.com/codeforscience/sciencefair#roadmap))
